docs/ruleset_functions/is_fluid_loop_purchased_heating.md

```diff
@@ -0,0 +1,28 @@
+# is_fluid_loop_purchased_heating  
+
+**Description:** Returns TRUE if the fluid loop associated with the heating system associated with the HVAC system is attached to an external purchased heating loop. Returns FALSE if this is not the case.   
+
+**Inputs:**  
+- **B-RMR**: To evaluate if the fluid loop associated with the heating system associated with the HVAC system is attached to an external purchased heating loop.   
+- **hvac_b.id**: The id of the hvac system to evaluate.  
+
+**Returns:**  
+- **is_fluid_loop_purchased_heating**: Returns TRUE if the fluid loop associated with the heating system associated with the HVAC system is attached to an external purchased heating loop. Returns FALSE if this is not the case.   
+ 
+**Function Call:** None  
+
+## Logic:   
+- Check if RMR is not modeled with external fluid source: `if NOT RMR.ASHRAE229.external_fluid_source: is_fluid_loop_purchased_heating = FALSE`
```

Comment:
Should be RulesetModelInstance.external_fluid_source. This applies to line 17 as well.

Reply:
Corrected